webapp/src/components/sidebar/SideBar.vue

```diff
@@ -14,66 +16,77 @@
 
       <!-- Mini Sidebar content-->
       <div class="flex flex-col items-center justify-center flex-1 space-y-4">
-        <SidebarButton to="/" text="Home"> <HomeIcon /> </SidebarButton>
+        <SidebarButton to="/" text="Home" id="home-icon">
+          <HomeIcon />
+        </SidebarButton>
 
         <SidebarButton to="/list" text="DISCOllaboratives">
-          <TasksIcon class="w-6 h-6" />
+          <TasksIcon class="w-6 h-6" id="task-icon" />
         </SidebarButton>
 
         <SidebarButton to="/create" text="Create a new DISCOllaborative">
-          <CreateIcon />
+          <CreateIcon id="create-icon" />
         </SidebarButton>
 
         <SidebarButton to="/evaluate" text="Evaluate models">
-          <EvaluateIcon />
-        </SidebarButton>
-
-        <SidebarButton to="/information" text="More on DISCO">
-          <InfoIcon />
+          <EvaluateIcon id="evaluate-icon" />
         </SidebarButton>
 
-        <SidebarButton 
-          to="" 
-          :text="currentTheme === 'light' ? 'Switch to Dark Mode' : 'Switch to Light Mode'" 
+        <SidebarButton
+          to=""
+          :text="
+            currentTheme === 'light'
+              ? 'Switch to Dark Mode'
+              : 'Switch to Light Mode'
+          "
           @click="toggleDarkMode"
         >
-          <MoonIcon v-if="currentTheme === 'light'"/>
-          <SunIcon v-else/>
+          <MoonIcon v-if="currentTheme === 'light'" class="theme-button" />
+          <SunIcon v-else class="theme-button" />
+        </SidebarButton>
+
+        <SidebarButton text="Show tutorial" to="" @click="showGuide">
```

Comment:
in my mind, a tutorial will take you fom beginning to end on the whole project, but theses drivers are rather helping with the current page

```suggestion
        <SidebarButton text="Show help" to="" @click="showGuide">
```